src/App.module.css

```
@@ -1,61 +1,1076 @@
+  --app-border-strong: #c7ced9;
+  --app-panel: #ffffff;
+  --app-sidebar: #f6f8fb;
+  --app-muted: #667085;
+  --app-text: #182230;
+  --app-blue: #246bce;
+  --app-blue-soft: #eaf2ff;
+  --app-hover: #f2f5f9;
+  --app-selected: #e8f1ff;
+  --app-error: #a33a3a;
+  --app-error-bg: #fff2f2;
+  width: 100%;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)